portfolio/src/main/webapp/gallery.html

```diff
@@ -9,14 +9,12 @@
 
 <body>
     <div id="content">
-     <div id="center">
-        <h1> Pictures of Jaden</h1>
```

Comment:
nit: Remove extra space before "P" in "Pictures". If you need space there to make the layout work, use margin or padding on that element